### DISCUSSION TITLE
GraphQL and Task timed out after 10.01 seconds

### DISCUSSION THREAD

**Discussion by smalloutside-admin** [vercel/vercel#5423] [General]
Hi,

I am seeing a lot of "Task timed out after 10.01 seconds" errors on the live site. The Vercel team has suggested that this could be down to code initialisation taking excessive time. I have also seen other help which suggests that Vercel puts the application in a "light sleep" mode. So putting two and two together, I think the issue we are facing is that the waking up from light sleep along with the time it takes to initialise a GraphQL call is taking in excess of 10 seconds and causing errors.

I have used sample code from Vercel site for using GraphQL, so not sure what else I can do. Has anyone faced this problem?

**Comment by stemmlerjs** [comment]
I've been getting this often on a few of my functions in production.
